scripts/outline_svg.js

```diff
@@ -0,0 +1,55 @@
+const { promises: fs } = require("fs");
+const outlineStroke = require("svg-outline-stroke");
+const { parse, stringify } = require("svgson");
+
+const inputDir = `./icons/`;
+const outputDir = `./converted_icons/`;
+
+async function init() {
+  try {
+    const files = await fs.readdir(inputDir);
+    for (let file of files) {
+      const icon = await fs.readFile(`${inputDir}${file}`);
+      const scaled = await parse(icon.toString(), {
+        transformNode: transformForward
+      });
+      const outlined = await outlineStroke(stringify(scaled));
+      const outlinedWithoutAttrs = await parse(outlined, {
+        transformNode: transformBackwards
+      });
+      await fs.writeFile(
+        `${outputDir}${file}`,
+        stringify(outlinedWithoutAttrs)
+      );
+    }
+  } catch (err) {
+    console.log(err);
+  }
+}
+
+init();
+
+function transformForward(node) {
+  if (node.name === "svg") {
+    return {
+      ...node,
+      attributes: {
+        ...node.attributes,
+        width: 800,
+        height: 800
```

Comment:
I think it'd better to use round binairy numbers for better scale factor.
icon: 24px * 40 = 960

```suggestion
        width: 960,
        height: 960
```